### ISSUE TITLE
Specifying a flag twice throws an error

### ISSUE THREAD

**Issue by hawkrives** [prettier/prettier#5369]
**Environments:**
- Prettier Version: 1.15.1
- Running Prettier via: CLI
- Runtime: node 11.1.0
- Operating System: macOS 10.14

**Steps to reproduce:**
```console
$ echo '{"name": "a-package"}' > ./package.json
$ prettier --tab-width 2 --use-tabs --tab-width 4 ./package.json
```

**Expected behavior:**
```console
$ prettier --tab-width 2 --use-tabs --tab-width 4 ./package.json
{
	"name": "a-package"
}
```

**Actual behavior:**
```console
$ prettier --tab-width 2 --use-tabs --tab-width 4 package.json
[error] Invalid --tab-width value. Expected an integer, but received null.
```

---

I happen to find myself in a situation where I need to override a CLI flag with another one, which appended to the invocation later. I cannot just remove the first flag, unfortunately.

What I would expect would be for Prettier to use the last value given for a flag, instead of trying to combine them into an array? (At least, I think that's what `minimist` is doing; I'm not sure where the error about "null" is coming from.)

**Comment by ikatyang** [comment]
The fix is available in [`1.15.2`](https://github.com/prettier/prettier/blob/master/CHANGELOG.md#1152).

**Comment by hawkrives** [comment]
Thanks so much!
